test/livebook/teams_test.exs

```diff
@@ -0,0 +1,92 @@
+defmodule Livebook.TeamsTest do
+  use Livebook.TeamsIntegrationCase, async: true
+
+  alias Livebook.Teams
+
+  describe "create_org/1" do
+    test "returns the device flow data to confirm the org creation" do
+      org = build(:org)
+
+      assert {:ok,
+              %{
+                "device_code" => _device_code,
+                "expires_in" => 300,
+                "id" => _org_id,
+                "user_code" => _user_code,
+                "verification_uri" => _verification_uri
+              }} = Teams.create_org(org)
+    end
+
+    test "returns changeset errors when data is invalid" do
+      assert {:error, errors} = Teams.create_org(build(:org, name: nil))
+      assert "can't be blank" in errors.name
+
+      assert {:error, errors} = Teams.create_org(build(:org, teams_key: nil))
+      assert "can't be blank" in errors.teams_key
+    end
+  end
+
+  describe "get_org_request_completion_data/1" do
+    test "returns the org data when it has been confirmed", %{node: node, user: user} do
+      org_request = :erpc.call(node, Hub.Integration, :create_org_request, [])
+      org_request = :erpc.call(node, Hub.Integration, :confirm_org_request, [org_request, user])
+
+      org =
+        build(:org,
+          id: org_request.id,
+          name: org_request.name,
+          teams_key: org_request.key_hash,
```

Comment:
These are different, no?

Reply:
Right, so in this case `org.teams_key` should be the actual key, while `org_request.key_hash` is its hash, no?

Reply:
Exactly!